src/coreComponents/finiteVolume/TwoPointFluxApproximation.cpp

```diff
@@ -862,16 +861,21 @@ void TwoPointFluxApproximation::computeBoundaryStencil( MeshLevel & mesh,
   ElementRegionManager::ElementViewAccessor< arrayView1d< integer const > > const elemGhostRank =
     elemManager.constructArrayViewAccessor< integer, 1 >( ObjectManagerBase::viewKeyStruct::ghostRankString() );
 
+  // TODO: can we look this up better?
+  string const & meshBodyName = mesh.getParent().getParent().getName();
+  arrayView1d< string const > const targetRegions = m_targetRegions.at( meshBodyName );
+  string const coeffModelNames = m_coefficientModelNames.at( meshBodyName )[0];
```

Comment:
Why do you only consider the first element of the list?

Reply:
They are all the same value....in the case of flow, it will be a string named "permeabilityNames", which is actually set in `FlowSolverBase`.  We will need to clean this concept up/out in a later PR.

Reply:
OK, great. Maybe should be have a list of the small pebbles in our shoes that we need to solve after this PR is merged?

Reply:
@francoishamon @CusiniM  are aware of this, and I think they have a plan.